src/Adapter/ContainerBuilder.php

```diff
@@ -216,7 +216,7 @@ private function loadServicesFromConfig(SfContainerBuilder $container)
      * Needs to be done as earlier as possible in application lifecycle. Unfortunately this can't
      * be done in a compiler pass because they are only executed on compilation and this needs to
      * be done at each container instanciation.
-
+     *
```

Comment:
I'm not sure why this additional blank line

Reply:
Fixed